config/rollup.js

```diff
@@ -20,6 +23,9 @@ const rollup = {
   plugins: [
     // eslint(), TODO: ES lint is throwing errors, it needs to be configured for ES6
     resolve(),
+    commonjs(),
+    vue(),
+    buble(),
```

Comment:
Import the necessary plugins for Rollup to recognize .vue files when they are imported in our scripts.